### PR TITLE
fix: Building block descriptions

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: src/components/BuildingBlocks.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const buildingBlocks = [
     href: '/collections',
     name: 'Collections',
     description:
-      'Learn about the conversation model and how to create, retrieve, update, delete, and list conversations.',
+      'A built in No-SQL store to quickly store and retrieve unstructure data.',
```
Comment (HomelessDinosaur) [inline src/components/BuildingBlocks.tsx:31]:
```suggestion
      'A built-in No-SQL store to quickly store and retrieve unstructured data.',
```

File: src/components/BuildingBlocks.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const buildingBlocks = [
     href: '/schedules',
     name: 'Schedules',
     description:
-      'Learn about the message model and how to create, retrieve, update, delete, and list messages.',
+      'Schedule actions easily with nitric using either simple rate expressions or cron.',
```
Comment (HomelessDinosaur) [inline src/components/BuildingBlocks.tsx:45]:
```suggestion
      'Schedule actions easily using either simple rate expressions or cron.',
```

Comment (HomelessDinosaur) [reply src/components/BuildingBlocks.tsx:45]: Reasoning is consistency with other descriptions

File: src/components/BuildingBlocks.tsx [hunk 3]
```diff
@@ -78,7 +78,7 @@ const buildingBlocks = [
     href: '/secrets',
     name: 'Secrets',
     description:
-      'Learn about the group model and how to create, retrieve, update, delete, and list groups.',
+      'Securely store secrets with nitrics secrets API',
```
Comment (HomelessDinosaur) [inline src/components/BuildingBlocks.tsx]:
```suggestion
      'Securely store, version, and access secret values',
```

File: src/components/BuildingBlocks.tsx [hunk 4]
```diff
@@ -67,7 +67,7 @@ const buildingBlocks = [
     href: '/messaging',
     name: 'Messages',
     description:
-      'Learn about the group model and how to create, retrieve, update, delete, and list groups.',
+      'Use Topics and Queues to create distributed applications',
```
Comment (HomelessDinosaur) [inline src/components/BuildingBlocks.tsx:70]:
```suggestion
      'Use Topics and Queues to create event-driven distributed applications',
```

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**